File: Data Structures & Algorithms/max-consecutive-ones/submission-5.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int i = 0;
        int max = 0;
        int current = 0;
        while (i < nums.size()){
            if (nums[i] == 1){
                current++;
            } else {
                current = 0;
            }

            if (current > max){
                max = current;
            }
            i++;
        }
        return max;
    }
};

File: Data Structures & Algorithms/contains-duplicate-ii/submission-5.py
class Solution:
    def containsNearbyDuplicate(self, nums: List[int], k: int) -> bool:
        for i in range(len(nums)-1):
            j = i + 1
            for j in range(len(nums)):
                if i != j and nums[i] == nums[j] and abs(i - j)<=k:
                    return True
        return False